sample/src/main/java/com/maxkeppeler/sample/custom_sheets_example/CustomSheet.kt

```diff
@@ -20,18 +20,54 @@ import android.content.Context
 import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
+import androidx.annotation.DrawableRes
 import androidx.annotation.StringRes
-import com.maxkeppeler.sheets.core.Sheet
-import com.maxkeppeler.sheets.core.PositiveListener
 import com.maxkeppeler.sample.databinding.BottomSheetsCustomBinding
+import com.maxkeppeler.sheets.core.PositiveListener
+import com.maxkeppeler.sheets.core.Sheet
 
 @Suppress("unused")
 class CustomSheet : Sheet() {
 
     override val dialogTag = "CustomSheet"
+    private var contentText: String? = null
 
     private lateinit var binding: BottomSheetsCustomBinding
 
+    companion object {
+        private const val STATE_CONTENT_TEXT = "state_content_text"
+    }
+
+
+    /**
+     * Set the content of the sheet.
+     *
+     * @param content The text for the content.
+     */
+    fun content(content: String) {
+        this.contentText = content
+    }
+
+    /**
+     * Set the content of the sheet.
+     *
+     * @param contentRes The String resource id for the content.
+     */
+    fun content(@StringRes contentRes: Int) {
+        this.contentText = windowContext.getString(contentRes)
+    }
+
+    /**
+     * Set the content of the sheet.
+     *
+     * @param contentRes Resource id for the format string
+     * @param formatArgs The format arguments that will be used for
+     *                   substitution.
+     */
+    fun content(@StringRes contentRes: Int, vararg formatArgs: Any?) {
+        this.contentText = windowContext.getString(contentRes, *formatArgs)
+    }
+
```

Comment:
I don't want to make the Custom Sheet to be a copy of the InfoSheet.

Reply:
There are some issues implementing the Custom View by just adding the core dependency to the project. Like couldn't implement the Positive onClick Listener.